index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ProjectManager</title>
+    <link rel="stylesheet" href="app.css"/>
+    <script src="dist/app.js" defer></script>
+</head>
+<body>
+    <template id="project-input">
+        <form>
+            <div class="form-control">
+                <label for="title">Title</label>
+                <input type="text" id="title"/>
+            </div>
+            <div class="form-control">
+                <label for="description">Description</label>
+                <textarea id="description" rows="3"></textarea>
+            </div>            
+            <div class="form-control">
+                <label for="people">people</label>
+                <input type="number" id="people" step="1" min="0" max="10"/>
+            </div>
+            <button type="submit">ADD Project</button>
+        </form>
+    </template>
+    <template id="single-project">
+        <li draggable="true">
+            <h2></h2>
```

Comment:
Why it is empty?

Reply:
Here the contents are coming from typescript  file